helpers.js

```diff
@@ -92,27 +92,21 @@ function fixClass(cls) {
   cls = cls.replace(/^(\.)/, "");
   // make other dots safe
   cls = cls.replace(/\\\./g, ".");
-  // remove extras at end
+  // remove > anything
+  cls = cls.replace(/\s?>\s?.*/, "");
+  // remove pseudo-elements (::)
+  cls = cls.replace(/::.*$/, "");
```

Comment:
There are no cases in TailwindCSS of `::` being used for anything other than pseudo-elements so it is safe to strip this out to the end. This would also take out any following pseudo-classes, but there are no cases of this in TailwindCSS at the moment.